src/navigation/BottomTabNavigator.tsx

```diff
@@ -25,14 +26,49 @@ const tabs = [
   { name: 'FEED' },
 ]
 
-export const BottomTabNavigator = ({ style }: ViewProps) => {
+const screenStyles = {
+  shadowOffset: { width: 0, height: 0 },
+  shadowColor: theme.colors.black,
+  shadowRadius: 10,
+  borderWidth: 0,
+  backgroundColor: '#0000',
```

Comment:
Why not use theme.colors for backgroundColor value?

Reply:
idk, i've just copied previous solution, I will change :D